plugins/experimental/ts_lua/ts_lua_misc.c

```diff
@@ -66,6 +71,20 @@ ts_lua_inject_misc_variables(lua_State *L)
 }
 
 static int
+ts_lua_get_process_id(lua_State *L)
+{
+  const char *s;
+  TSUuid process = TSProcessUuidGet();
+  if (process) {
+    s = TSUuidStringGet(process);
```

Comment:
None of these calls fail. IMHO it is fine to ``lua_error`` or ``ink_release_assert`` rather than attempting error handling.